package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "4.0.23",
+  "version": "4.1.0",
```

Comment:
No particular reason to update other than 23 is pretty high for a featureset such as registrations or business lookup.